frontend/components/multi-select/multi-select.js

```diff
@@ -64,6 +67,19 @@ MultiSelect.prototype.handleToggleAllButtonChanged = function () {
   )
 }
 
+MultiSelect.prototype.handleDisabledButtons = function (checkCount) {
```

Comment:
I'm not convinced this is the correct place to put this logic. I'm split whether the requirement to disable the "edit allocation" button if two or more prisoners are selected is an implementation specific requirement or generic functionality of the multi-select component.

However, disabling a button when multiple items are selected seems like it could be useful to other implementations so I've decided to add it here to the component code instead of doing something implementation specific.

Reply:
Agreed. Fine here, for me.